test/CreatorToken.t.sol

```diff
@@ -17,6 +17,8 @@ contract CreatorTokenTest is Test {
   string PAY_TOKEN_NAME = "Payment Token";
   string PAY_TOKEN_SYMBOL = "PAY";
   uint256 BASE_PAY_AMOUNT = 1e18; // Because our test token has 18 decimals
+  uint256 constant CREATOR_FEE = 700; // 7%
+  uint256 constant ADMIN_FEE = 300; // 3%
```

Comment:
Let's pull out an issue to think about how to run the tests with varying fee values.

Reply:
Created this issue: https://github.com/showtime-xyz/creator-tokens/issues/20